### PR TITLE
Introduce play 2.8 + scala 2.13

### PR DESCRIPTION
**Pull request by jlogeart**
DO NOT MERGE

The goal of this PR is just to show the diff and allow for reviews.

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -22,8 +22,8 @@ lazy val root = project
       "org.apache.commons" % "commons-io" % "1.3.2",
       "org.scalatestplus.play" %% "scalatestplus-play" % "4.0.3" % Test,
       specs2 % Test,
-      compilerPlugin("com.github.ghik" %% "silencer-plugin" % "1.4.2"),
-      "com.github.ghik" %% "silencer-lib" % "1.4.2" % Provided,
+      compilerPlugin("com.github.ghik" %% "silencer-plugin" % "1.4.4" cross CrossVersion.full),
```
Comment (juwen-z) [inline build.sbt]: Do we want to cross build or keep separate branches for `play26` and `play28`?

Comment (jlogeart) [reply build.sbt]: We want to keep separate branches.

Play 2.6 is not cross compiled in scala 2.12. Therefore, as we are migrating, we made the decision to migrate to scala 2.13 and play 2.8 at the same time.

As indicated in the description, this branch will not be merged as all other libraries in the tree first need to be updated. Once all libraries have a play 2.8 version, and the server have been updated, we will merge all branches. No play2.6 will be left.

**Comment by jlogeart** [comment]
Following the reviews, proceeding with https://github.com/flowcommerce/lib-play/pull/344

**Comment by gheine** [comment]
Should we keep building two version of this lib, for play 2.6 and 2.8, for a while?

**Comment by jlogeart** [comment]
@gheine Yes, as we have been doing. 
That is why it says `DO NOT MERGE` 😄

**Pull request merged**